components/Viewer/Generic/Mesh/Edges/Color.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+    <ContextMenuItem v-bind="itemProps">
+      <template #tooltip>Edges color</template>
+      <template #btn>
+        <slot name="btn"></slot>
+      </template>
+      <template #options>
+        <v-card>
+          <v-card-text class="justify-center">
+            <v-switch v-model="visibility" inset label="Visibility" />
+            <template v-if="visibility">
+              <v-combobox
+                v-model="select"
+                :items="styles"
+                label="Select color style"
+              ></v-combobox>
+              <v-color-picker
+                v-if="select === styles[0]"
+                ref="action"
+                v-model="color"
+                flat
+                canvas-height="100"
+                hide-inputs
+                width="200"
+              ></v-color-picker>
+            </template>
+          </v-card-text>
+        </v-card>
+      </template>
+    </ContextMenuItem>
+  </template>
+  
+  <script setup>
+  
+  const props = defineProps({
+    itemProps: { type: Object, required: true },
+  });
+  
+  console.log("ViewerGenericMeshEdgesColor props", props.itemProps);
+  
+  const dataStyleStore = useDataStyleStore();
+  const tree_view_store = use_treeview_store();
+  
+  const id = toRef(() => props.itemProps.id);
+  console.log("id", id.value);
+  const meta_data = computed(() => {
+    return tree_view_store.idMetaData(id.value);
+  });
+  
+  console.log("meta_data", meta_data);
+  
+  const visibility = computed({
+    get() {
+      return dataStyleStore.edgesVisibility(id.value);
+    },
+    set(newValue) {
+      dataStyleStore.setEdgesVisibility(id.value, newValue);
+    },
+  });
+  const color = computed({
+    get() {
+      return dataStyleStore.edgesConstantColor(id.value);
+    },
+    set(newValue) {
+      dataStyleStore.setEdgesConstantColor(id.value, newValue);
+    },
+  });
+  const styles = ["Constant"];
+  const storeStyles = ["constant"];
+  const select = computed({
+    get() {
+      const active = dataStyleStore.edgesActiveColoring(id.value);
+      for (let i = 0; i < styles.length; i++) {
+        if (active === storeStyles[i]) {
+          return styles[i];
+        }
+      }
+      return "";
```

Comment:
```suggestion
      return "Select a coloring";
```
ou un truc du genre ?